.travis.yml

```diff
@@ -10,8 +13,10 @@ env:
   - TEST_SUITE=./spec/build_spec.cr
   - TEST_SUITE=./spec/amber
 
-script: 
-  - docker-compose run --service-ports spec bash -c "cd /opt/amber && crystal spec $TEST_SUITE -D run_build_tests"
+script:
+  - crystal spec
+  - bin/ameba
```

Comment:
You should run it in a docker together with `crystal spec`. Maybe @drujensen can suggest the right approach?

Reply:
Done! 👍